test/driver.js

```diff
@@ -468,12 +472,13 @@ var Driver = (function DriverClosure() {
               var textLayerContext = textLayerCanvas.getContext('2d');
               textLayerContext.clearRect(0, 0,
                 textLayerCanvas.width, textLayerCanvas.height);
+              var enhanceText = task.id.includes('enhance') ? true : false;
```

Comment:
Good, we had the same thought :)

I would personally prefer `enhance_text_selection` to make it clear what is being enhanced, much like the already used `enhanceTextSelection` variable in the code.